CONTRIBUTING.md

```diff
@@ -33,8 +36,7 @@ The Docker Bench has the main script called `docker-bench-security.sh`. This is
 The tests are split in 6 different files:
 
 ```sh
-✗  docker-bench-security git:(master) ✗ tree tests
-tests
+✗ tests
```

Comment:
This makes it look like `tests` is a command, where in reality it is a directory.